src/main/java/net/minecraftforge/oredict/OreDictionary.java

```diff
@@ -106,6 +106,12 @@ private static void initVanillaEntries()
             registerOre("fenceGateWood", Blocks.JUNGLE_FENCE_GATE);
             registerOre("fenceGateWood", Blocks.DARK_OAK_FENCE_GATE);
             registerOre("fenceGateWood", Blocks.ACACIA_FENCE_GATE);
+            registerOre("doorWood", new ItemStack(Items.ACACIA_DOOR));
+            registerOre("doorWood", new ItemStack(Items.BIRCH_DOOR));
+            registerOre("doorWood", new ItemStack(Items.DARK_OAK_DOOR));
+            registerOre("doorWood", new ItemStack(Items.OAK_DOOR));
+            registerOre("doorWood", new ItemStack(Items.JUNGLE_DOOR));
+            registerOre("doorWood", new ItemStack(Items.SPRUCE_DOOR));
```

Comment:
Hmm, why do you wrap these items in item stacks while you can pass in the item as an argument?